frontend/src/pages/answer/fullKg/__mocks__/simulation.worker.js

```diff
@@ -0,0 +1,12 @@
+class Worker {
+  constructor(stringUrl) {
+    this.url = stringUrl;
+    this.onmessage = () => {};
+  }
+
+  postMessage(msg) {
+    this.onmessage(msg);
+  }
+}
+
+export default Worker;
```

Comment:
I suspect that I could use some documentation here. Why do we need this? Does posting a message actually do nothing?

Reply:
Updated in 0bf32c925c56cdabe83847a950f3ca76781ff742

Reply:
I feel like anywhere the mocked thing behaves differently from the actual thing, we should include documentation about how and why. Here my question is why `postMessage` does nothing by default. The answer may be tied in with how the tests work, but they live elsewhere.

Reply:
Updated in 0e1fb37bdd4814fcc3bc834e8c187eb42d9bcb6a